app/src/test/java/mollie/tictactoe/MobilePlayerTest.java

```diff
@@ -23,7 +23,7 @@ public void returns0ForLocation() throws Exception {
     }
 
     @Test
-    public void testPlayerType() throws Exception {
+    public void returnsCorrectPlayerType() throws Exception {
         assertEquals(MobilePlayer.class, player.playerType());
```

Comment:
Not really sure this test has value. It's not testing any business logic. It is a bit like saying `assert(ArrayList.class, List.class)` which we wouldn't really do. Is there a reason why you are checking this here?